courses/fundamentals_of_ada/230_interfacing_with_c++.rst

```diff
@@ -236,13 +236,22 @@ Complex Data Types
 Unions
 --------
 
+* C/C++ `union`
+    
+   .. code:: C++
+    
+      union Rec {
+         int A;
+         float B;
+      };
+     
 * C unions can be bound using the `Unchecked_Union` aspect
 * These types must have a mutable discriminant for convention purpose, which doesn't exist at run-time
 
    - All checks based on its value are removed - safety loss
    - It cannot be manually accessed
 
-* Ada View
+* Ada implementation of a C `union`
```

Comment:
"Ada-style C union" ?

Reply:
No, because it's not a C union - it's just how Ada implements one